packages/sdk-vue/src/createFusionAuth/createFusionAuth.test.ts

```diff
@@ -35,7 +35,11 @@ describe('createFusionAuth', () => {
   });
 
   it('Fetches userInfo', async () => {
-    const user = { given_name: 'JSON', family_name: 'Bourne' };
+    const user = {
+      given_name: 'JSON',
+      family_name: 'Bourne',
+      customTrait: 'additional info',
```

Comment:
Is there a reason for this change? I believe we can use this test to cover what we're implementing here, but I'm not sure this is doing it. [Here's an example](https://github.com/FusionAuth/fusionauth-javascript-sdk/blob/4a215cae7e3efe4dbf7934d7c94b1cc7ef541749/packages/sdk-angular/projects/fusionauth-angular-sdk/src/lib/fusion-auth.service.spec.ts#L46)